lib/msal-common/src/app/module/AuthorizationCodeModule.ts

```diff
@@ -84,15 +84,20 @@ export class AuthorizationCodeModule extends AuthModule {
             );
 
             // Check for SSO
+            let adalIdToken: IdToken;
```

Comment:
should we really be naming things with `adal`?

Reply:
This is for SSOing to an ADAL session, so yes, in this case.

Reply:
As Jason said, this is a one-off specifically for ADAL, so doesn't get used otherwise.